src/math/functions/Vec2Func.js

```diff
@@ -233,6 +233,26 @@ export function lerp(out, a, b, t) {
     return out;
 }
 
+/**
+ * Performs a frame rate independant, linear interpolation between two vec2's
+ *
+ * @param {vec2} out the receiving vector
+ * @param {vec2} a the first operand
+ * @param {vec2} b the second operand
+ * @param {Number} decay decay constant for interpolation
```

Comment:
Could you please add `Useful range approx 1 to 25, from slow to fast`, just to give a bit more of an idea of how to use this? Same for Vec3Func please